#!/usr/bin/env python
PACKAGE = "motion_turn"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("double_param",    double_t,    0, "Angle to turn", 90, -180, 180)
gen.add("bool_param",   bool_t,   0, "A Boolean parameter",  False)

exit(gen.generate(PACKAGE, "TurnXY", "turning"))
